Pollo/Views/PollBuilder/OnboardingView.swift

```diff
@@ -0,0 +1,528 @@
+                self.askQuestionBorder.alpha = 1
+                self.firstOptionView.alpha = 1
+                self.firstCorrectView.alpha = 1
+                self.firstOptionLabel.alpha = 1
+                self.secondOptionView.alpha = 1
+                self.secondCorrectView.alpha = 1
+                self.secondOptionLabel.alpha = 1
+                self.addOptionView.alpha = 1
+                self.plusLabel.alpha = 1
+                self.addOptionLabel.alpha = 1
+                self.animatedLabel.alpha = 1
+            }) { completed in
+                if completed {
+                    self.isTransitioning = true
+                }
+            }
+        case .createQuestion:
+            stage = .autofillChoices
+            animatedLabel.text = getAnimatedLabelText()
+            firstOptionLabel.text = firstOptionTextAutofilled
+            secondOptionLabel.text = secondOptionTextAutofilled
+            animatedLabel.textAlignment = .right
+            UIView.animate(withDuration: animationDuration, animations: {
+                self.askQuestionLabel.alpha = 0
+                self.askQuestionBorder.alpha = 0
+                self.addOptionView.alpha = 0
+                self.plusLabel.alpha = 0
+                self.addOptionLabel.alpha = 0
+                self.buttonView.alpha = 1
+                self.buttonLabel.alpha = 1
+                self.animatedLabel.snp.remakeConstraints { remake in
+                    remake.leading.equalToSuperview().offset(self.getAnimatedLabelLeadingOffset())
+                    remake.trailing.equalTo(self.firstOptionView).inset(self.getAnimatedLabelTrailingInset())
+                    remake.bottom.equalTo(self.safeAreaLayoutGuide.snp.bottom).offset(-self.animatedLabelBottomOffset)
+                }
+                self.layoutIfNeeded()
+            }) { completed in
+                if completed {
+                    self.isTransitioning = true
+                }
+            }
+        case .autofillChoices:
+            stage = .quizMode
+            UIView.animate(withDuration: animationDuration, animations: {
+                self.firstOptionView.alpha = 0
+                self.firstOptionLabel.alpha = 0
+                self.secondOptionView.alpha = 0
+                self.secondOptionLabel.alpha = 0
+                self.buttonView.alpha = 0
+                self.buttonLabel.alpha = 0
+                self.animatedLabel.snp.remakeConstraints { remake in
+                remake.leading.equalTo(self.firstCorrectView.snp.trailing).offset(self.getAnimatedLabelLeadingOffset())
+                    remake.top.equalTo(self.firstCorrectView).offset(-self.animatedLabelBottomSmallOffset)
+                    remake.trailing.equalToSuperview().inset(self.getAnimatedLabelTrailingInset())
+                }
+                self.layoutIfNeeded()
+            }) { completed in
+                if completed {
+                    self.animatedLabel.text = self.getAnimatedLabelText()
```

Comment:
The label's transition (text change and left alignment) looks pretty jumpy, especially with slow animations turned on. Generally, let's change the text _before_ animating for consistency with the other transitions.
Also, some of the jumpiness is likely because the label's frame is changing size and cuts off the text for a brief second, especially when it switches from right to left alignment:
![image](https://user-images.githubusercontent.com/34046832/68883958-1d2ab300-06e0-11ea-9e8d-33ac91d2901b.png)

Something you could try is to hardcode the size of the `animatedLabel`'s frame so that it doesn't resize every time the text changes content or alignment. Alternatively you can play around with using a UIView.transition every time you change animatedLabel to make it look smoother?
Ex:
`UIView.transition(with: animatedLabel, duration: 0.25, options: .transitionCrossDissolve, animations: {
...
}`

Reply:
Sounds good. I'll try that. Thanks!

Reply:
@jackthmp I made it so that the text changes before the animation occurs. Does this still look jumpy to you?